index.js

```diff
@@ -0,0 +1,121 @@
+window.onload = function () {
+    const canvas = document.getElementById("canvas");
+    const context = canvas.getContext("2d");
+    const BB = canvas.getBoundingClientRect();
+
+    canvas.width = window.innerWidth * 0.8;
```

Comment:
Can be done in CSS instead?

Reply:
No, it's not possible. CSS styles only stretch the canvas, rather than expand